cypress/integration/portal-dashboard-smoke-test.spec.js

```diff
@@ -0,0 +1,15 @@
+
+context("Portal Dashboard UI",()=>{
+    before(()=>{
+        cy.visit("/?portal-dashboard");
+    });
+    describe('progress',()=>{
+        it('verify progress legend loads',()=>{
+            const progress=["Completed","In progress", "Not yet started"];
+            cy.get('[data-test=progress-legend]').should('be.visible');
+            progress.forEach((prog)=>{
+                cy.get('[data-test="'+prog+'-legend"]').should('be.visible');
+            });
+        });
+    });
+});
```

Comment:
Could we add an extra newline here?  Kirk pushed me to be consistent with doing this.  Also thanks for adding this model.  I'm going to reuse it to add tests for the other components.